eng/common/pipelines/templates/steps/cosmos-emulator.yml

```diff
@@ -9,92 +9,8 @@ steps:
       Write-Host "Target Dir: $targetDir"
       msiexec /a ${{ parameters.EmulatorMsiUrl }} TARGETDIR=$targetDir /qn | wait-process
     displayName: Download and Extract Public Cosmos DB Emulator
-  - powershell: |
```

Comment:
Looks like you have some outdated changes.

Reply:
I am testing my changes with the tools change. Will revert the changes.